src/components/WebexInMeeting/WebexInMeeting.css

```diff
@@ -0,0 +1,22 @@
+.in-meeting {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.in-meeting .remote-media {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.in-meeting .local-media {
+  position: absolute;
+  bottom: 25%;
+  right: 3%;
+  width: 20%;
+  height: 20%;
+  z-index: 1;
+  background: white;
```

Comment:
Why are making the background white?

Reply:
the background is transparent and thats not what we want.

Reply:
so I set it to white.

Reply:
the localMedia component needs some styling works. I removed it form this PR. there will be a ticket for styling the localMedia